docs/source/polars-cloud/cli.md

```diff
@@ -5,51 +5,20 @@ with polars cloud resources from the terminal.
 
 ```bash
 pc --help
-```
 
-```
-usage: pc [-h] [-v] [-V] {login,workspace,compute} ...
+usage: pc [-h] [-v] [-V] {authenticate,setup,login,organization,workspace,compute} ...
 
 positional arguments:
-{login,workspace,compute}
-login               Authenticate with Polars Cloud by logging in through the browser
-workspace           Manage Polars Cloud workspaces.
-compute             Manage Polars Cloud compute clusters.
-
-options:
--h, --help            show this help message and exit
--v, --verbose         Output debug logging messages.
--V, --version         Display the version of the Polars Cloud client.
-```
-
-### Authentication
-
-You can authenticate with Polars Cloud from the CLI using
-
-```bash
-pc login
-```
-
-This refreshes your access token and saves it to disk.
-
-### Workspaces
-
-Create and setup a new workspace
-
-```bash
-pc workspace setup
+  {authenticate,setup,login,organization,workspace,compute}
+    authenticate        Authenticate with Polars Cloud by loading stored credentials or otherwise logging in through the browser.
+    setup               Set up an organization and workspace to quickly run queries. Ideal to get started with Polars Cloud.
+    login               Authenticate with Polars Cloud by logging in through the browser.
+    organization        Manage Polars Cloud organizations.
+    workspace           Manage Polars Cloud workspaces.
+    compute             Manage Polars Cloud compute clusters.
+
+optional arguments:
+  -h, --help            show this help message and exit.
+  -v, --verbose         Output debug logging messages.
+  -V, --version         Display the version of the Polars Cloud client.
```

Comment:
I don't think we should explain every single command here unless they start getting to be very complex. Right now the explanation of the commands would be one liners with basically the same info as the help command.